README.md

```diff
@@ -1 +1,46 @@
-# Ruby-project1
\ No newline at end of file
+# Capstone Project NTML & CSS
+
+This is the Ruby project #1, from the Microverse's main curriculum and aims to build two algorithms similar to the sorting method, without using those methods.
+
+![screenshot](./assets/Ruby-pro_1.png)
+
+## Built With
+
+- RUBY
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Live Server plugin for Visual Studio Code
```

Comment:
Do we really need the Live Server plugin in a Ruby project?

- [ ] Please add actual "Prerequisites" for your project.